README.md
ExercicioEstagio


xampp/RELEASENOTES
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2019-01-18] XAMPP for Windows 5.6.40-0
This version of XAMPP contains the following software releases:
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-12-14] XAMPP for Windows 5.6.39-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.39
   - Updated MariaDB to 10.1.37
   - Updated OpenSSL 1.0.2p
   - Updated phpMyAdmin 4.8.4

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-09-25] XAMPP for Windows 5.6.38-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.38
   - Updated MariaDB to 10.1.36
   - Updated phpMyAdmin 4.8.3

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-08-08] XAMPP for Windows 5.6.37
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.37
   - Updated MariaDB to 10.1.34
   - Updated phpMyAdmin 4.8.2

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-05-11] XAMPP for Windows 5.6.36-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.36
   - Updated MariaDB to 10.1.32
   - Updated OpenSSL 1.0.2o
   - Updated phpMyAdmin 4.8.0.1

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-04-12] XAMPP for Windows 5.6.35-0
This version of XAMPP contains the following software releases:
   - Updated Apache to 2.4.33
   - Updated PHP to 5.6.35
   - Updated phpMyAdmin 4.8.0

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-03-15] XAMPP for Windows 5.6.34-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.34
   - Updated MariaDB to 10.1.31
   - Updated phpMyAdmin 4.7.9

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-01-19] XAMPP for Windows 5.6.33-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.33
   - Updated MariaDB to 10.1.30
   - Updated phpMyAdmin 4.7.7
   - Updated OpenSSL to 1.0.2n
   - Updated Apache to 2.4.29

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2017-11-10] XAMPP for Windows 5.6.32-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 5.6.32
   - Updated OpenSSL to 1.0.2m
   - Updated Apache to 2.4.28
   - Updated MariaDB to 10.1.28
   - Updated phpMyAdmin to 4.7.4

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2017-07-11] XAMPP for Windows 5.6.31-0
This version of XAMPP includes:
    - Updated MariaDB to 10.1.25
    - Updated phpMyAdmin to 4.7.0
    - Updated PHP to 5.6.31
    - Updated Apache to 2.4.26
    - Add UTF8 and Unicode Properties support to PCRE

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2017-02-01] XAMPP for Windows 5.6.30-0
This version of XAMPP includes:
    - Updated PHP to 5.6.30
    - Updated MariaDB to 10.1.21 (Security release CVE-2016-6664 CVE-2017-3238)
    - Updated Apache to 2.4.25

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-12-21] XAMPP for Windows 5.6.28-1
This version of XAMPP includes:
   - Fixed XAMPP Control Panel Overflow DOS

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-11-15] XAMPP for Windows 5.6.28-0
This version of XAMPP includes:
   - Updated PHP to 5.6.28

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-11-10] XAMPP for Windows 5.6.24-1
This version of XAMPP includes:
    - Updated MariaDB to 10.1.19 (CVE-2016-6663 CVE-2016-6664)

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-08-31] XAMPP for Windows 5.6.24-0
This version of XAMPP includes:
    - Updated PHP to 5.6.24
    - Updated MariaDB to 10.1.16
    - Updated Apache to 2.4.23

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-06-28] XAMPP for Windows 5.6.23-0
This version of XAMPP includes:
    - Updated PHP to 5.6.23

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-05-12] XAMPP for Windows 5.6.21-0
This version of XAMPP includes:
    - Updated PHP to 5.6.21 (Security release CVE-2016-3074)
    - Updated OpenSSL to 1.0.2h (Security release CVE-2016-2108 CVE-2016-2107)

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-04-21] XAMPP for Windows 5.6.20
This version of XAMPP includes:
    - Updated PHP to 5.6.20
    - Updated MariaDB to 10.1.13

-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-03-04] XAMPP for Windows 5.6.19-0
This version of XAMPP includes:
    - Updated PHP to 5.6.19

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2016-02-24] XAMPP for Windows 5.6.15-2
This version of XAMPP includes:
    - Fixed issue with phpMyAdmin

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2015-12-06] XAMPP for Windows 5.6.15-0
This version of XAMPP includes:
    - Updated PHP to 5.6.15
    - OpenSSL 1.0.1q
    - Fixed phpMyAdmin issues with pma database

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2015-11-13] XAMPP for Windows 5.6.14-3
This version of XAMPP includes:
    - Updated MariaDB to 10.1.8
    - Updated phpMyAdmin 4.5.1
    - Fixed issue starting/stopping MariaDB from the Control Panel

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2015-10-19] XAMPP for Windows 5.6.14
This version of XAMPP includes:
    - Updated Apache to 2.4.17
    - Updated PHP to 5.6.14
    - Added MariaDB 10.0.17
    - Updated phpMyAdmin 4.5.0.2


xampp/phpMyAdmin/tmp/twig/04/04377191582b8e0ae95010c2dfd0c11db876ca5022d217748d516633bc17e30f.php
<?php

/* display/export/options_output.twig */
class __TwigTemplate_8eafb4f546e963a9514eeb8eb45c4a8b07b9b20206ec442f615a32ed38aba748 extends Twig_Template
{
    public function __construct(Twig_Environment $env)
    {
        parent::__construct($env);

        $this->parent = false;

        $this->blocks = array(
        );
    }

    protected function doDisplay(array $context, array $blocks = array())
    {
        // line 1
        echo "<div class=\"exportoptions\" id=\"output\">
    <h3>";
        // line 2
        echo _gettext("Output:");
        echo "</h3>
    <ul id=\"ul_output\">
        <li>
            <input type=\"checkbox\" id=\"btn_alias_config\"";
        // line 5
        echo (((isset($context["has_aliases"]) ? $context["has_aliases"] : null)) ? (" checked") : (""));
        echo ">
            <label for=\"btn_alias_config\">
                ";
        // line 7
        echo _gettext("Rename exported databases/tables/columns");
        // line 8
        echo "            </label>
        </li>

        ";
        // line 11
        if (((isset($context["export_type"]) ? $context["export_type"] : null) != "server")) {
            // line 12
            echo "            <li>
                <input type=\"checkbox\" name=\"lock_tables\"
                    value=\"something\" id=\"checkbox_lock_tables\"";
            // line 15
            echo (((( !(isset($context["repopulate"]) ? $context["repopulate"] : null) && (isset($context["is_checked_lock_tables"]) ? $context["is_checked_lock_tables"] : null)) || (isset($context["lock_tables"]) ? $context["lock_tables"] : null))) ? (" checked") : (""));
            echo ">
                <label for=\"checkbox_lock_tables\">
                    ";
            // line 17
            echo sprintf(_gettext("Use %s statement"), "<code>LOCK TABLES</code>");
            echo "
                </label>
            </li>
        ";
        }
        // line 21
        echo "
        <li>
            <input type=\"radio\" name=\"output_format\" value=\"sendit\" id=\"radio_dump_asfile\"";
        // line 24
        echo ((( !(isset($context["repopulate"]) ? $context["repopulate"] : null) && (isset($context["is_checked_asfile"]) ? $context["is_checked_asfile"] : null))) ? (" checked") : (""));
        echo ">
            <label for=\"radio_dump_asfile\">
                ";
        // line 26
        echo _gettext("Save output to a file");
        // line 27
        echo "            </label>
            <ul id=\"ul_save_asfile\">
                ";
        // line 29
        if ( !twig_test_empty((isset($context["save_dir"]) ? $context["save_dir"] : null))) {
            // line 30
            echo "                    ";
            echo (isset($context["options_output_save_dir"]) ? $context["options_output_save_dir"] : null);
            echo "
                ";
        }
        // line 32
        echo "
                ";
        // line 33
        echo (isset($context["options_output_format"]) ? $context["options_output_format"] : null);
        echo "

                ";
        // line 35
        if ((isset($context["is_encoding_supported"]) ? $context["is_encoding_supported"] : null)) {
            // line 36
            echo "                    ";
            echo (isset($context["options_output_charset"]) ? $context["options_output_charset"] : null);
            echo "
                ";
        }
        // line 38
        echo "
                ";
        // line 39
        echo (isset($context["options_output_compression"]) ? $context["options_output_compression"] : null);
        echo "

                ";
        // line 41
        if ((((isset($context["export_type"]) ? $context["export_type"] : null) == "server") || ((isset($context["export_type"]) ? $context["export_type"] : null) == "database"))) {
            // line 42
            echo "                    ";
            echo (isset($context["options_output_separate_files"]) ? $context["options_output_separate_files"] : null);
            echo "
                ";
        }
        // line 44
        echo "            </ul>
        </li>

        ";
        // line 47
        echo (isset($context["options_output_radio"]) ? $context["options_output_radio"] : null);
        echo "
    </ul>

    <label for=\"maxsize\">";
        // line 51
        echo sprintf(_gettext("Skip tables larger than %s MiB"), "</label><input type=\"text\" id=\"maxsize\" name=\"maxsize\" size=\"4\">");
        // line 53
        echo "
</div>
";
    }

    public function getTemplateName()
    {
        return "display/export/options_output.twig";
    }

    public function isTraitable()
    {
        return false;
    }

    public function getDebugInfo()
    {
        return array (  126 => 53,  124 => 51,  118 => 47,  113 => 44,  107 => 42,  105 => 41,  100 => 39,  97 => 38,  91 => 36,  89 => 35,  84 => 33,  81 => 32,  75 => 30,  73 => 29,  69 => 27,  67 => 26,  62 => 24,  58 => 21,  51 => 17,  46 => 15,  42 => 12,  40 => 11,  35 => 8,  33 => 7,  28 => 5,  22 => 2,  19 => 1,);
    }

    /** @deprecated since 1.27 (to be removed in 2.0). Use getSourceContext() instead */
    public function getSource()
    {
        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.27 and will be removed in 2.0. Use getSourceContext() instead.', E_USER_DEPRECATED);

        return $this->getSourceContext()->getCode();
    }

    public function getSourceContext()
    {
        return new Twig_Source("", "display/export/options_output.twig", "C:\\xampp\\phpMyAdmin\\templates\\display\\export\\options_output.twig");
    }
}
